### ISSUE TITLE
Issues building with GHC 9.0.1

### ISSUE THREAD

**Issue by yobson** [schernichkin/partial-isomorphisms#4]
Hello!

I was building this library with GHC 9.0.1 (it is a dependency of a XML library we are trying out) and I got the following error:

```
Building library for partial-isomorphisms-0.2.2.1..
[1 of 6] Compiling Control.Isomorphism.Partial.Unsafe ( src/Control/Isomorphism/Partial/Unsafe.hs, dist/build/Control/Isomorphism/Partial/Unsafe.o, dist/build/Control/Isomorphism/Partial/Unsafe.dyn_o )
[2 of 6] Compiling Control.Isomorphism.Partial.TH ( src/Control/Isomorphism/Partial/TH.hs, dist/build/Control/Isomorphism/Partial/TH.o, dist/build/Control/Isomorphism/Partial/TH.dyn_o )

src/Control/Isomorphism/Partial/TH.hs:37:30: error:
    • Expecting one more argument to ‘TyVarBndr’
      Expected a type, but ‘TyVarBndr’ has kind ‘* -> *’
    • In the type ‘[TyVarBndr]’
      In the definition of data constructor ‘DecInfo’
      In the data declaration for ‘DecInfo’
   |
37 | data DecInfo = DecInfo Type [TyVarBndr] [Con]
   |                              ^^^^^^^^^
cabal: Failed to build partial-isomorphisms-0.2.2.1 (which is required by
exe:licence-check from licence-check-0.1.0.0). See the build log above for
details.
```

All the best

James

**Comment by schernichkin** [comment]
Fixed in 0.2.3.0: https://hackage.haskell.org/package/partial-isomorphisms-0.2.3.0
Please let me know if is it OK to close the issue.

**Comment by yobson** [comment]
I can confirm that it's fixed.

Thanks so much!
